crates/node/src/lib.rs

```diff
@@ -82,6 +82,16 @@ pub struct Scanner {
   scanner: tailwindcss_oxide::Scanner,
 }
 
+#[derive(Debug, Clone)]
+#[napi(object)]
+pub struct CandidateWithPosition {
+  // The candidate string
+  pub candidate: String,
+
+  // The position of the candidate inside the content file
+  pub position: i64,
```

Comment:
If you use `///` then they become dockblocks and will be visible when hovering over `position` somewhere else in your codebase.
```suggestion
  /// The candidate string
  pub candidate: String,

  /// The position of the candidate inside the content file
  pub position: i64,
```